<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 0, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="head_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Quiz App</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
border:none;
border-bottom: 2px solid rgba(105, 118, 132, 255);
color: rgb(190, 190, 190);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="sur_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
border:none;
border-bottom: 2px solid rgba(105, 118, 132, 255);
color: rgb(190, 190, 190);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
border:none;
border-bottom: 2px solid rgba(105, 118, 132, 255);
color: rgb(190, 190, 190);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>General Topics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sport</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Science</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Math</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>History</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>English Level Test</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
